index.html

```diff
@@ -2,23 +2,40 @@
 title: Path of Building Community Fork
 description: A powerful build planner for Path of Exile
 ---
-<section class="hero">
+<section class="hero" style="padding-top: 0px;">
 	<div class="text-container">
 		<h1 class="editable app-title">Path of Building</h1>
-		<h2 class="app-subtitle">Community Fork</h2>
-		<p class="subtext editable">A powerful build planner for Path of Exile</p>
-		<div>
-			<img src="{{ site.baseurl }}/images/pob_overview.png" alt="Screenshot" />
-		</div>
-		<div class="cta button alt">
-			<a href="https://github.com/PathOfBuildingCommunity/PathOfBuilding/releases/latest/download/PathOfBuildingCommunity-Setup.exe">
-				<span class="social-icon">{% include social-icon.html icon="GitHub" %}</span>
-				Download!
-			</a>
+		<h2 class="app-subtitle">A powerful build planner for Path of Exile</h2>
+		<div class="flex">
+			<div class="cta">
+				<div class="cta">
+					<img src="{{ site.baseurl }}/images/pob2_overview.png" alt="Screenshot" />
+				</div>
+				<div class="cta button alt">
+					<a href="https://github.com/PathOfBuildingCommunity/PathOfBuilding-PoE2/releases/latest/download/PathOfBuildingCommunity-PoE2-Setup.exe ">
+						<img class="social-icon" src="{{ site.baseurl }}/images/poe2_icon.png" alt="Path of Exile 1 logo"/>
+						Download POB2 for POE2!
```

Comment:
Change these to not use acronyms
```suggestion
						Download for Path of Exile 2!
```

Reply:
Added in https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/pull/19/commits/e235590cfb39d395101e2fbdc4768b3390504c71

Reply:
Oh, I actually meant just remove the "Path of Building [2]" altogether.  It gets a little cramped and wordy with both terms there, and I think the "Path of Building" is redundant considering it's at the top of the page (see original website how it doesn't have it either)

Reply:
I'll make it just `Download for Path of Exile [2]`. Local had some concerns regarding clarity of which button goes to which version so i wanted to make it clear.

Reply:
I think now that we have the icon of the game itself it's a little clearer which is which